README.md

```diff
@@ -1,72 +1,29 @@
-## Nixer spring-plugin by Crossword Cybersecurity
+# Nixer Spring Plugin ![](https://github.com/nixer-io/nixer-spring-plugin/workflows/Java%20CI/badge.svg)
 
-A plugin for [Spring](https://github.com/spring-projects/spring-framework) utilising Nixer's protection technologies by [Crossword Cybersecurity](https://www.crosswordcybersecurity.com/). The plugin provides protection against credential stuffing attacks to your Spring web application. 
+Nixer plugin for [Spring](https://github.com/spring-projects/spring-framework) framework for dealing with credential stuffing attacks.
 
-## Credential Stuffing attack
+For overview and tutorials visit [our documentation website](https://nixer-io.github.io/).
 
-Credential Stuffing is a peculiar attack on web applications. Technically it is a very basic and easy to understand attack closely related to brute force techniques, yet it is frighteningly effective. 
- 
-To learn more about credential stuffing take a look at [OWASP definition](https://www.owasp.org/index.php/Credential_stuffing) or our [introductory article into the subject](https://medium.com/@jbron/credential-stuffing-how-its-done-and-what-to-do-with-it-57ad66302ce2).
+## Distribution
 
-## Credential Stuffing protection as a Spring library - why?
+The plugin is distributed through [Maven Central](https://search.maven.org/search?q=g:io.nixer).
```

Comment:
Can we see stats on maven.org. Mvnrepository seems better because it shows number of usages and stats.